collect_app/src/main/res/xml/analytics_tracker.xml

```diff
@@ -1,5 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Thanks, @shobhitagarwal1612! @cdwuestner I believe this file is implicitly used by the analytics library. The Android Studio tools are very handy but unfortunately they're not totally bullet-proof.